.gitattributes

```diff
@@ -3,5 +3,5 @@ travis.zip.enc merge=ours
 README.md merge=ours
 src/app/customHelpers.js merge=ours
 src/app/components/media/MediaStatus.js merge=ours
-test/spec/status_spec.rb merge=ours
```

Comment:
oh it was for Travis... yes and let's do this work when we merge back into `bridge-web` for the final unification